calculatorFrame.java

```diff
@@ -111,191 +104,192 @@ public void init() {
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Eight button
-		else if (e.getSource() == eight) {
-			if (flag == false){
+		else if(e.getSource() == eight) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "8";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "8";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Nine button
-		else if (e.getSource() == nine) {
-			if (flag == false){
+		else if(e.getSource() == nine) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "9";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			}else if(flag == true) {
 				secondNumString += "9";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Zero button
-		else if (e.getSource() == zero){
-			if (flag == false){
+		else if(e.getSource() == zero) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "0";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "0";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//DECIMAL
-		else if (e.getSource() == decimal){
-			if (flag == false && decimalFlag == false){
+		else if(e.getSource() == decimal) {
+			if(flag == false && decimalFlag == false) {
 				System.out.println("Decimal");
 				firstNumString += ".";
 				decimalFlag = true;
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true && decimalFlag == false){
+			} else if(flag == true && decimalFlag == false) {
 				secondNumString += ".";
 				decimalFlag = true;
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else{
+			} else {
 				textArea.setText(textArea.getText() + "\n" +"Error. A decimal point has already been used." + "\n");
 			}
 		}
-		
+
 		//ADDITION
-		else if (e.getSource() == plus){
+		else if(e.getSource() == plus) {
 			System.out.println("FirstNum=" + firstNumString + "\n" + "SecondNum=" + secondNumString);
-			if(duplicateOperatorFlag == true){
+
+			if(duplicateOperatorFlag == true) {
 				textArea.setText(textArea.getText() + "\n" + "Operator + has been ignored" + "\n");
-			}else if(flag ==  true && duplicateOperatorFlag == false && plusFlag == true){
+			} else if(flag ==  true && duplicateOperatorFlag == false && plusFlag == true) {
```

Comment:
Do you need a `duplicateOperatorFlag` or can you just use the last operator entered if consecutive operators are entered?